versions.yaml

```diff
@@ -75,7 +75,7 @@ assets:
       url: "https://github.com/intel/cloud-hypervisor"
       uscan-url: >-
         https://github.com/intel/cloud-hypervisor/tags.*/v?(\d\S+)\.tar\.gz
-      version: "ca97385da5ebb798"
+      version: "ee528ae80836e9d59ff50d124ae356ed1b07002a"
```

Comment:
I wonder if we should treat this as a re-vendor? If so, would be good to include the changes between these commits in one of the commits in this PR.

Reply:
good idea, added the changes in the commit